tests/cephadm/test_bootstrap.py

```diff
@@ -371,6 +392,42 @@ def verify_ceph_public_ssh_key(cls, ssh_key_path):
     log.info("Ceph SSH public key file found")
 
 
+def verify_ceph_custom_public_private_ssh_key(cls, ssh_key_path, ssh_pri_key_path):
```

Comment:
would it make sense to rename to `verify_custom_ssh_keys`?

Reply:
can we add this file existence verification in `validate_ssh_public_key` and `validate_ssh_private_key` functions?

Reply:
Added the file existence verification in validate functions. Removed the method verify_ceph_custom_public_private_ssh_key